core-compose/src/androidTest/java/com/squareup/workflow/ui/compose/RenderAsStateTest.kt

```diff
@@ -41,6 +40,7 @@ import okio.ByteString.Companion.decodeBase64
 import org.junit.Rule
 import org.junit.Test
 import org.junit.runner.RunWith
+import androidx.ui.test.waitForIdle as waitForIdle1
```

Comment:
Is there any clashes if we use `waitForIdle`?

Reply:
Oops, I think this was a fat-finger.